galaxy/worker/importers/base.py

```diff
@@ -101,7 +111,7 @@ def _update_readme(self, content):
 
     def _log_create_content(self, content_id, is_created):
         action = 'Created new' if is_created else 'Found'
-        self.log.debug(
+        LOG.debug(
```

Comment:
Why this change?

Reply:
Bad. Will revert that. It's what I had to do in order to see the output. The self.log.debug() messages don't surface in the API response for some reason, and thus don't appear on the UI.